### ISSUE TITLE
1.1.1l+quic "no-quic" builds fail unit tests

### ISSUE THREAD

**Issue by tmshort** [quictls/openssl#46]
When building 1.1.1l+quic:

```
./config no-quic
make -sj4
make tests
```

A number of tests are failing. They don't fail when compiled with QUIC.

**Comment by tmshort** [comment]
```
Test Summary Report
-------------------
../test/recipes/70-test_tls13kexmodes.t          (Wstat: 1536 Tests: 11 Failed: 6)
  Failed tests:  4-5, 7-10
  Non-zero exit status: 6
../test/recipes/70-test_tls13messages.t          (Wstat: 512 Tests: 17 Failed: 2)
  Failed tests:  2, 16
  Non-zero exit status: 2
../test/recipes/70-test_tls13psk.t               (Wstat: 256 Tests: 5 Failed: 1)
  Failed test:  5
  Non-zero exit status: 1
../test/recipes/80-test_ssl_new.t                (Wstat: 1792 Tests: 29 Failed: 7)
  Failed tests:  2, 5, 9-10, 12-13, 27
  Non-zero exit status: 7
../test/recipes/80-test_ssl_old.t                (Wstat: 256 Tests: 6 Failed: 1)
  Failed test:  4
  Non-zero exit status: 1
../test/recipes/90-test_sslapi.t                 (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
../test/recipes/90-test_tls13ccs.t               (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
../test/recipes/99-test_fuzz.t                   (Wstat: 256 Tests: 11 Failed: 1)
  Failed test:  8
  Non-zero exit status: 1

**Comment by tmshort** [comment]
Fixed via #49
